webhook-notifications_5def901604286364bc92af37.md

```diff
@@ -0,0 +1,283 @@
+Semaphore has integrated Webhook based notifications that are delivered on
```

Comment:
The word "intgrated" is commonly used to refer to an integration with an outside service. Use:

> Semaphore has webhook based notifications that are delivered ...